rss2email/email.py

```diff
@@ -254,18 +254,33 @@ def lmtp_send(recipient, message, config=None, section='DEFAULT'):
 def imap_send(message, config=None, section='DEFAULT'):
     if config is None:
         config = _config.CONFIG
-    server = config.get(section, 'imap-server')
-    port = config.getint(section, 'imap-port')
-    _LOG.debug('sending message to {}:{}'.format(server, port))
-    ssl = config.getboolean(section, 'imap-ssl')
-    if ssl:
+    server = config.get(section, "imap-server")
```

Comment:
Why did you change everything to use double quotes?

Reply:
I normally automatically run black... will update this now to single quotes

Reply:
should be fixed now